Assets/Scripts/Utilities/MaterialChanger.cs

```diff
@@ -7,6 +7,11 @@ public class MaterialChanger : MonoBehaviour
     public Material[] cachedMaterial;
     public MeshRenderer cachedRenderer;
 
+    //todo: make sure the main materials are here, Unity's material manipulation is not really nice.
+    public  Material Palette01            = Resources.Load<Material>("Materials/Pallete01.mat");
+    public  Material Palette05            = Resources.Load<Material>("Materials/Pallete05.mat");
+    public  Material Palette05Emission    = Resources.Load<Material>("Materials/Pallete05 Emission.mat");
```

Comment:
You added these, but are not using them

Reply:
yes didn't wanted to change the material folder to resources right now, when we implement more item usage it will be more important